IAdapter.cs

```diff
@@ -0,0 +1,15 @@
+using static System.Console;
+namespace adapter_dp
+{
+    public class IAdapter
```

Comment:
`IAdapter` should be an interface

Reply:
> **Amazing job Younes!! 👏👏**
> You have used a great idea to showcase how the Adapter design pattern solves problems! Thank you for your creative work.
> But I noticed that your code is very similar to "Software Design Patterns C#.NET by K.P.P. Suneth Pathirana | 6. Adapter Design Pattern".
> Thank you.
> 
> > don't forget to check the comments on your code.

Thank you Norah.

Yes, my solution is based on that article. I was looking to practice not find creative use for the dp like other homework.

Reply:
Good approach! but keep in mind that in future you have to reference any resource you have used.

Reply:
Yes, Ma'am!